xformers/ops/fmha/common.py

```diff
@@ -99,6 +99,43 @@ def normalize_bmhk(self) -> Tuple[int, ...]:
             self.value = self.value.unsqueeze(2)
         return output_shape
 
+    def validate_bmhk(self) -> None:
+        qkv = (self.query, self.key, self.value)
+        if tuple(x.ndim for x in qkv) != (4, 4, 4):
+            raise ValueError(
+                f"Query/Key/Value should have BMHK format.\n"
```

Comment:
This error could be misleading. The user can pass a `BMK` tensor as well, and will be confused as to why this mentions that only 4d inputs are valid